packages/patternfly-4/react-table/src/components/Table/base/resolve-row-key.ts

```diff
@@ -8,6 +8,9 @@
 import { isArray } from 'lodash';
 import { RowType, RowKeyType } from './types';
 
+/**
+ *
+ */
```

Comment:
You can add some simple JSDoc instead of leaving it blank.

Reply:
Done